test/unit/spec/narrative_core/upload/importSetup-spec.js

```diff
@@ -286,138 +286,268 @@ define([
+                        file2 = `${'b'.repeat(1500)}.csv`,
+                        dataType = 'assembly',
+                        response1 = {
+                            types: {
+                                [dataType]: { some: 'data' },
+                            },
+                            files: {
+                                [dataType]: { file: file1, tab: null },
+                            },
                         },
-                        files: {
-                            [testCase.dataType]: {
-                                file: fileName,
-                                tab: null,
+                        response2 = {
+                            types: {
+                                [dataType]: { some: 'more_data' },
+                            },
+                            files: {
+                                [dataType]: { file: file2, tab: null },
                             },
+                        };
+
+                    jasmine.Ajax.stubRequest(
+                        `${stagingServiceUrl}/bulk_specification/?files=${file1}`
+                    ).andReturn({
+                        status: 200,
+                        statusText: 'ok',
+                        contentType: 'text/plain',
+                        responseHeaders: '',
+                        responseText: JSON.stringify(response1),
+                    });
+                    jasmine.Ajax.stubRequest(
+                        `${stagingServiceUrl}/bulk_specification/?files=${file2}`
+                    ).andReturn({
+                        status: 200,
+                        statusText: 'ok',
+                        contentType: 'text/plain',
+                        responseHeaders: '',
+                        responseText: JSON.stringify(response2),
+                    });
+
+                    const importInputs = [
+                        {
+                            name: file1,
+                            type: 'import_specification',
                         },
-                    };
-                    await testWithExpectedImportErrors([fileName], stagingResponse);
+                        {
+                            name: file2,
+                            type: 'import_specification',
+                        },
+                    ];
+
+                    let error;
+                    try {
+                        await ImportSetup.setupImportCells(importInputs);
+                    } catch (e) {
+                        error = e;
+                    }
+                    // make sure we get the right list, and that the fileErrors
+                    // look like : { file1: [error], file2: [error] }
+                    expect(error).toEqual(jasmine.any(Errors.ImportSetupError));
+                    [file1, file2].forEach((fileName) => {
+                        expect(error.fileErrors[fileName]).toEqual(jasmine.any(Array));
+                    });
```

Comment:
is it possible to test the error structure more completely? Do you know what is going to be in `error.fileErrors`?

Reply:
Ah, I thought I committed that change. Ugh, will do so in a few minutes.